contractcourt/channel_arbitrator.go

```diff
@@ -342,7 +342,7 @@ type ChannelArbitrator struct {
 	// htlcUpdates is a channel that is sent upon with new updates from the
 	// active channel. Each time a new commitment state is accepted, the
 	// set of HTLC's on the new state should be sent across this channel.
-	htlcUpdates <-chan *ContractUpdate
+	htlcUpdates chan *contractUpdateSignal
```

Comment:
I think it's better to unify the naming a bit. Since we already have `signalUpdates chan *signalUpdateMsg`, maybe use `htlcUpdates chan *htlcUpdatesMsg`?

Reply:
I agree about name unification, I just don't think this is the pr to do it in. I tried to change as little as possible here. Plus it's delivering `ContractUpdate` pointers so I find `htlcUpdatesMsg` a little odd